core/ft.go

```diff
@@ -0,0 +1,559 @@
+package core
+
+import (
+	"bytes"
+	"errors"
+	"fmt"
+	"math"
+	"strconv"
+	"strings"
+	"time"
+
+	"github.com/rubixchain/rubixgoplatform/block"
+	"github.com/rubixchain/rubixgoplatform/contract"
+	"github.com/rubixchain/rubixgoplatform/core/model"
+	"github.com/rubixchain/rubixgoplatform/core/wallet"
+	"github.com/rubixchain/rubixgoplatform/rac"
+	"github.com/rubixchain/rubixgoplatform/util"
+	"github.com/rubixchain/rubixgoplatform/wrapper/uuid"
+)
+
+func (c *Core) CreateFTs(reqID string, did string, ftcount int, ftname string, wholeToken float64) {
+	err := c.createFTs(reqID, ftname, ftcount, wholeToken, did)
+	br := model.BasicResponse{
+		Status:  true,
+		Message: "FT created successfully",
+	}
+	if err != nil {
+		br.Status = false
+		br.Message = err.Error()
+	}
+	channel := c.GetWebReq(reqID)
+	if channel == nil {
+		c.log.Error("Failed to get did channels")
+		return
+	}
+	channel.OutChan <- &br
+}
+
+func (c *Core) createFTs(reqID string, FTName string, numFTs int, numWholeTokens float64, did string) error {
```

Comment:
any reason for float64 to have variable name as numWholeTokens

Reply:
Changed to int.